app/views/shared/steps/_broken-links-summary.html.erb

```diff
@@ -0,0 +1,2 @@
+<span class="glyphicon glyphicon-warning-sign" aria-hidden="true"></span>
+<strong><%= step.broken_links.count %></strong> broken link<%= "(s)" if step.broken_links.count > 1 %> found
```

Comment:
What is the reasoning behind using `<strong>` here, to only wrap the number?

Reply:
cc @DilwoarH

Reply:
Not sure why this was done, maybe we should remove it or wrap it around the whole text

Reply:
@DilwoarH I'll wrap the whole thing and see what it looks like.